krabs/krabs/trace.hpp

```diff
@@ -123,6 +123,30 @@ namespace krabs {
          */
         void set_trace_properties(const PEVENT_TRACE_PROPERTIES properties);
 
+        /**
+         * <summary>
+         * Configures trace session settings.
+         * See https://docs.microsoft.com/en-us/windows/win32/api/evntrace/nf-evntrace-tracesetinformation
+         * for more information.
+         * </summary>
+         * <example>
+         *    krabs::trace trace;
+         *    // Adjust SE_SYSTEM_PROFILE_NAME token privilege through AdjustTokenPrivileges(...)
+         *    // to enable stack tracing (not done in this example). Then:
+         *    STACK_TRACING_EVENT_ID event_id = {0};
+         *    event_id.EventGuid = krabs::guids::perf_info;
+         *    event_id.Type = 46; // SampleProfile
+         *    trace_.set_trace_information(TraceStackTracingInfo, &event_id, sizeof(STACK_TRACING_EVENT_ID));
+         *    krabs::kernel_provider stack_walk_provider(EVENT_TRACE_FLAG_PROFILE, krabs::guids::stack_walk);
+         *    trace_.enable(stack_walk_provider);
+         *    trace.start();
+         * </example>
+         */
+        void set_trace_information(
+            TRACE_INFO_CLASS informationClass,
```

Comment:
let's use snake_case for these parameter names

Reply:
Done.